pkg/api/handlers/compat/containers.go

```diff
@@ -403,6 +403,26 @@ func LibpodToContainerJSON(l *libpod.Container, sz bool) (*types.ContainerJSON,
 		state.Status = define.ContainerStateCreated.String()
 	}
 
+	state.Health = &types.Health{}
+	state.Health.Status = inspect.State.Healthcheck.Status
+	state.Health.FailingStreak = inspect.State.Healthcheck.FailingStreak
+
+	//item := define.HealthCheckLog{}
+
+	var res *types.HealthcheckResult = &types.HealthcheckResult{}
```

Comment:
This looks a little dubious - why is it being created only once? I think every time the for loop below runs, you're overwriting the contents, and appending the same entry to the array again?

I would move this inside the loop for safety